packages/react-components/src/components/MicrophoneButton.tsx

```diff
@@ -35,11 +89,52 @@ export interface MicrophoneButtonProps extends ControlBarButtonProps {
    * Maps directly to the `onClick` property.
    */
   onToggleMicrophone?: () => Promise<void>;
-
+  /* @conditional-compile-remove-from(stable) meeting-composite control-bar-split-buttons */
+  /**
+   * Available microphones for selection
+   */
+  microphones?: OptionsDevice[];
+  /* @conditional-compile-remove-from(stable) meeting-composite control-bar-split-buttons */
+  /**
+   * Available speakers for selection
+   */
+  speakers?: OptionsDevice[];
+  /* @conditional-compile-remove-from(stable) meeting-composite control-bar-split-buttons */
+  /**
+   * Microphone that is shown as currently selected
+   */
+  selectedMicrophone?: OptionsDevice;
+  /* @conditional-compile-remove-from(stable) meeting-composite control-bar-split-buttons */
+  /**
+   * Speaker that is shown as currently selected
+   */
+  selectedSpeaker?: OptionsDevice;
+  /* @conditional-compile-remove-from(stable) meeting-composite control-bar-split-buttons */
+  /**
+   * Callback when a microphone is selected
+   */
+  onSelectMicrophone?: (device: OptionsDevice) => Promise<void>;
+  /* @conditional-compile-remove-from(stable) meeting-composite control-bar-split-buttons */
+  /**
+   * Speaker when a speaker is selected
+   */
+  onSelectSpeaker?: (device: OptionsDevice) => Promise<void>;
+  /* @conditional-compile-remove-from(stable) meeting-composite control-bar-split-buttons */
+  /**
+   * Whether to use a {@link SplitButton} with a {@link IContextualMenu} for device selection.
+   *
+   * default: false
+   */
+  showDeviceSelectionMenu?: boolean;
```

Comment:
nit: show implies an action to make it shown or hidden
could this instead be:
`enableDeviceSelectionMenu` or `showDeviceSelectionSplitButton`

Reply:
I like `enableDeviceSelectionMenu`. Done.